src/libstd/io/stdio.rs

```diff
@@ -141,8 +141,7 @@ fn handle_ebadf<T>(r: io::Result<T>, default: T) -> io::Result<T> {
 ///
 /// Each handle is a shared reference to a global buffer of input data to this
 /// process. A handle can be `lock`'d to gain full access to [`BufRead`] methods
-/// (e.g. `.lines()`). Writes to this handle are otherwise locked with respect
-/// to other writes.
```

Comment:
I think this may have been intended to mention reads, so perhaps it could say "Reads to this handle are otherwise locked with respect to other reads."?